api-v3-beta.rst

```diff
@@ -0,0 +1,62 @@
+.. post:: August 8, 2019
+   :tags: api, feature, reference
+   :author: Manuel
+   :location: BCN
+
+.. meta::
+   :description lang=en:
+
+      Announcing API v3 Beta as invite-only status. Help us to make it *even* better!
+
+========================
+ Announcing API v3 Beta
+========================
+
+In the last months, we have been working on making our API better.
+Considering the limitations of our current REST API v2,
+we decided to make a bigger step forward and create our new shiny API v3 with better and bigger features,
+putting the focus on the user cases we heard about from existing users.
+
+Our new API v3 has some big differences compared to the old one that make it more user-friendly and useful.
+
+These differences includes:
+
+* Authenticated only
+* Token based authentication
+* Read and Write
+* Project's slug resource based
+
+Allowing us to build new good features on top of them, like:
```

Comment:
I like the suggestion. It's clearer.

Although, I wouldn't use "will" since it's possible to do all these queries currently.

> Some examples of new queries and actions that are possible with the API v3 are:

What do you think?